contents/dotfiles/Dotfiles.md

```diff
@@ -0,0 +1,218 @@
+<frontmatter>
+  title: Introduction to Dotfiles
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Dotfiles
+
+**Author(s): [Tiu Wee Han](https://github.com/tiuweehan)**<br>
+
+### What are Dotfiles?
+Dotfiles are plain text configuration files on Unix like systems (e.g. MacOS, Linux, BSD). Dotfiles store settings of almost every application, service and tool running on your system. These files control the behavior of applications from boot to termination and everything in between.
+
+<pic src="dotfiles-logo.png" alt="Dotfiles Logo" width="45%">
+  
+  <sub>_Figure 1. Dotfiles Logo_ [(source)](https://www.twilio.com/blog/using-dotfiles-productivity-bootstrap-systems)</sub>
+
+</pic>
+
+Some common examples of dotfiles include:
+- `.profile`, `.bashrc`, `.zshrc`: scripts that run when a Shell is started. 
+- `.nanorc`, `.vimrc`, `.emacs.d`: dotfiles for configuring text editors like nano, vim and emacs respectively.
+- `.gitconfig`, `.gitignore`: dotfiles containing git configurations.
+- `.ssh/config`: dotfiles containing SSH configurations.
+
```

Comment:
These seems to be sub points of what are dot files. maybe can arrange the points as 
`dot files can be used to`
-SSH configuration (eg)
-git configuration(eg)
etc

Reply:
That's a good suggestion, I will reorder the files and their usages